help/en/html/hardware/XPressNet/index.shtml

```diff
@@ -23,16 +23,16 @@
       <h1>Hardware Support: XpressNet</h1>
 
       <p>XpressNet is a Command Bus developed by Lenz and used by: <a href=
-      "http://www.atlasrr.com">Atlas</a>, <a href="https://uk.hornby.com">Hornby</a>, <a href=
-      "http://www.lenz.com">Lenz</a>, <a href="http://www.opendcc.de/index_e.html">OpenDCC
-      (DIY)</a>, <a href="http://usuaris.tinet.cat/fmco/home_en.htm">Paco Canada (DIY)</a>,
-      <a href="https://www.roco.cc/en/home/index.html">Roco</a> (some models), <a href=
-      "http://www.viessmann-modell.com/">Viessmann</a> and ZTC Controls.</p>
+      "https://shop.atlasrr.com/">Atlas</a>, <a href="https://uk.hornby.com">Hornby</a>, <a href=
+      "https://www.lenz-elektronik.de/index.php">Lenz</a>, <a href="https://www.opendcc.de/index_e.html">OpenDCC
+      (DIY)</a>, <a href="https://usuaris.tinet.cat/fmco/home_en.htm">Paco Canada (DIY)</a>,
+      <a href="https://www.roco.cc/ren/">Roco</a> (some models), <a href=
+      "https://viessmann-modell.com/">Viessmann</a> and ZTC Controls.</p>
       <!-- https://www.roco.cc/en/home/index.html -->
       <!-- http://www.roco.com -->
       https://dccwiki.com/XpressNet_Protocol https://uk.hornby.com
-      <p>The XpressNet specification (available from the <a href=
-      "http://www.lenzusa.com/techinfo/index.htm">Lenz website</a>) specifies the electrical
+      <p>The XpressNet specification ( formerly available from the 
+      http://www.lenzusa.com/techinfo/index.htm Lenz website ) specifies the electrical
       communications standards and command set for devices to communicate with the command
       station.</p>
```

Comment:
See this link:
https://wiki.rocrail.net/lib/exe/fetch.php?media=xpressnet:xpressnet-v2.pdf

Reply:
Thanks, does updated text scan ok?

Reply:
Yes-